Frameworks/CoreText/CTFontDescriptor.mm

```diff
@@ -111,21 +176,28 @@ CTFontDescriptorRef CTFontDescriptorCreateMatchingFontDescriptor(CTFontDescripto
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFDictionaryRef CTFontDescriptorCopyAttributes(CTFontDescriptorRef descriptor) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    CF_OBJC_FUNCDISPATCHV(CTFontDescriptorGetTypeID(), CFDictionaryRef, (UIFontDescriptor*)descriptor, fontAttributes);
+    __CFGenericValidateType(descriptor, CTFontDescriptorGetTypeID());
+    return CFDictionaryCreateCopy(CFGetAllocator(descriptor->_dictionary), descriptor->_dictionary);
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CFTypeRef CTFontDescriptorCopyAttribute(CTFontDescriptorRef descriptor, CFStringRef attribute) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    CF_OBJC_FUNCDISPATCHV(CTFontDescriptorGetTypeID(), CFTypeRef, (UIFontDescriptor*)descriptor, objectForKey
+                          : static_cast<NSString*>(attribute));
+    __CFGenericValidateType(descriptor, CTFontDescriptorGetTypeID());
+
+    // Use CFRetain instead of a true copy here
+    //  - A generic 'CFCopy' is not available, as not all types are copyable
```

Comment:
So it does... seems difficult to find though, so I'm not gonna invest the effort right now x___x

---
In reply to: [79066713](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79066713) [](ancestors = 79066713)